src/serviceWorker.js

```diff
@@ -11,17 +11,17 @@
 // opt-in, read https://bit.ly/CRA-PWA
 
 const isLocalhost = Boolean(
-  window.location.hostname === 'localhost' ||
+  window.location.hostname === "localhost" ||
```

Comment:
Why was this file edited?

Reply:
Got it. Just `git checkout serviceWorker.js`. We **don't** need to commit unnecessary files please.